
########################## CREATE REFERENCE ####################################  

### [START___REFERENCE___02/19/23 02:59:10 PM]
[W::fai_get_val] Reference HLA:HLA05880 not found in FASTA file, returning empty sequence
[faidx] Failed to fetch sequence in HLA:HLA05880
### [COMPLETE___REFERENCE___02/19/23 02:59:11 PM]

########################## RUN SCHLACOUNT ####################################  

### [START___SCHLACOUNT___02/19/23 02:59:12 PM]
### Starting  scHLAcount at 02/19/23 02:59:12 PM
Failed with error: Input "/labs/khatrilab/solomonb/covid/isb/scHLAcount/bam/INCOV098-BL_I4.bam" does not exist
### [COMPLETE___SCHLACOUNT___02/19/23 02:59:12 PM]
